### DISCUSSION TITLE
创建有状态服务器报错： in serverless mode, secret is required

### DISCUSSION THREAD

**Discussion by lauda0** [chanify/chanify#135] [Q&A]
./chanify serve --host=192.168.0.107 --port=9005 --name=w --datapath=~/.chanify --endpoint=http://192.168.0.107:9005

用了文档里的命令，提示报错如下：
2022/11/20 02:38:19 Launching service...
2022/11/20 02:38:19 Register user enabled
2022/11/20 02:38:19 Init service failed: in serverless mode, secret is required

难道是我的姿势不对？和文档上的命令一样的，没找到特别指定有状态模式的参数

**Comment by lauda0** [comment]
看源码原来是要指定dburl 好吧..

**Reply by GitXin** [reply]
也可以不指定 dburl，只指定 data 目录就可以，不过必须先创建 data 目录，不存在的话也会提示 secret is required

**Comment by wizjin** [comment]
你好，可以配置dburl来使用MySQL，也可以通过配置data目录来使用SQLite。
